packages/website/package.json

```diff
@@ -18,7 +18,7 @@
   "author": "Fabio Berger",
   "license": "Apache-2.0",
   "dependencies": {
-    "0x.js": "0xproject/0x.js/packages/0x.js#0x.js@0.27.1",
+    "0x.js": "^0.27.2",
```

Comment:
Mh, is  this temporary?

Reply:
I guess once we publish the new version, we can revert this to a hard-coded version.